source/Sashimi/Sashimi.csproj

```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Portable.BouncyCastle" Version="1.8.4" />
-    <PackageReference Include="Sashimi.Server.Contracts" Version="10.0.0" />
```

Comment:
This version is being downgraded back to the 9.x line. The only [addition in 10.x](https://github.com/OctopusDeploy/Sashimi/compare/9.0.2...octopusdeploy:10.0.0) is an extra `feedId` parameter in the `ICreateTargetServiceMessageHandler` interface. But in the 2021.1 release, [the calling code](https://github.com/OctopusDeploy/OctopusDeploy/blob/release/2021.1/source/Octopus.Server/Orchestration/ServerTasks/Deploy/ActionDispatch/ServiceMessageHandlers/CreateDeploymentTargetOrchestrator.cs#L265) does not provide that parameter.